climpred/prediction.py

```diff
@@ -701,13 +702,15 @@ def compute_perfect_model(ds,
     if metric in [_pearson_r, _rmse, _mse, _mae]:
         forecast, reference = comparison(ds, supervector_dim)
         res = metric(forecast, reference, dim=supervector_dim)
-        return res
     # perfect-model only metrics
     elif metric in [_nmae, _nrmse, _nmse, _ppp, _uacc]:
         res = metric(ds, control, comparison, running, reference_period)
-        return res
     else:
         raise ValueError('specify metric argument')
+    time_size = ds.time.size
```

Comment:
These lines here are breaking the notebook currently for `bootstrap_perfect_model`. Trying to debug.

```python
ValueError: cannot add coordinates with new dimensions to a DataArray
```

Reply:
Only seems to occur when `compute_perfect_model` is called from `bootstrap_perfect_model`. It's actually erroring on line 712: `res['time'] = np.arange(1, 1 + time_size)`. Removing these lines now so the notebook will compile prior to merge. You can open a quick PR to reinstate them if they are needed for something.

Reply:
Just commented them out for now, actually.